deploy-as-code/helm/charts/works/backend/attendance/values.yaml

```diff
@@ -1,7 +1,7 @@
 # Common Labels
 labels:
   app: "attendance"
-  group: "digit-works"
+  group: "works"
```

Comment:
Change namespace to egov, for common or registry services will be deployed in the egov namespace